controller/api/util/api_utils.go

```diff
@@ -55,11 +55,12 @@ var (
 // StatsBaseRequestParams contains parameters that are used to build requests
 // for metrics data.  This includes requests to StatSummary and TopRoutes.
 type StatsBaseRequestParams struct {
-	TimeWindow    string
-	Namespace     string
-	ResourceType  string
-	ResourceName  string
-	AllNamespaces bool
+	TimeWindow     string
+	Namespace      string
+	ResourceType   string
+	ResourceName   string
+	AllNamespaces  bool
+	ExcludeFromAll []string
```

Comment:
TIOLI: Maybe it is possible to validate the contents of this slice. Namely can we return an error if any of the to be excluded resources is not actually a valid resource name in order to avoid future headscratchers?

Reply:
Good idea, will do it, thanks 👍